### ISSUE TITLE
Option to hide specific namespaces

### ISSUE THREAD

**Issue by Suika** [hydrusnetwork/hydrus#887]
Add the ability to not displace certain namespaces. The same way as namespaces are applied, but do the opposite.
Since the color of a namespace is checked against whatever was defined in the "tag presentation", I think adding a check box "don't display this namespace" in the namespace editor would suffice.
And when the color for a namespace is read, also read if the namespace should be displayed.

**Comment by TheElo** [comment]
You can hide tags and namespaces in the display options.

**Comment by Zweibach** [comment]
@TheElo that's not what's being asked for here even if the OP is badly formulated. It's hiding the namespace of a tag. So `clothing:red dress` will display as `red dress`, not hiding the whole namespace with all its tags.

**Comment by luni3359** [comment]
For those that want to disable namespaces showing globally, it's on Options → Tag presentation → Show namespaces

**Comment by Zweibach** [comment]
Yes, but this is asking for disabling *specific* namespaces, not all of them.
